apps/web/components/ManufacturerInformation/ManufacturerInformation.vue

```diff
@@ -9,7 +9,9 @@
       <p v-if="manufacturerInfo.logo">
         <NuxtImg :src="manufacturerInfo.logo" :alt="t('manufacturer.logoAlt')" loading="lazy" />
       </p>
-      <p v-if="manufacturerInfo.externalName">{{ manufacturerInfo.externalName }}</p>
+      <p v-if="manufacturerInfo.name">{{ manufacturerInfo.name }}</p>
+      <p v-else-if="manufacturerInfo.externalName">{{ manufacturerInfo.externalName }}</p>
+      <p v-else-if="manufacturerInfo.legalName">{{ manufacturerInfo.legalName }}</p>
```

Comment:
```suggestion
      <p v-if="manufacturerInfo.legalName">{{ manufacturerInfo.legalName }}</p>
```

This always needs to be displayed right ?